### ISSUE TITLE
How to simplify the scene.

### ISSUE THREAD

**Issue by DangBowen-Bell** [ZhengyiLuo/EmbodiedPose#1]
Hi, thank you for your great work. I have some questions about how to simplify the scene.

In the paper, you say you simplify the scene semi-automatically by first selecting key points in the mesh vertices and then creating rectangular cuboids and cylinders. 

I want to know how to select the key points and create the cuboids. Are these operations done in MeshLab or Unity3D? I would appreciate it if you can give me some more instructions about the procedure. 

Looking forward to your reply!

**Comment by ZhengyiLuo** [comment]
The modification process is as follows: 

Given the scanned point cloud of the scenes, we first identify the objects (part of the point cloud) that we wish to simplify. For each object, we manually select the geometries we want to use to approximate it (e.g. cylinders and cuboids for tables). Then, for each segment of the objects (such as armrests and legs of chairs), we used a script to find the bounding box of the point clouds and calculate the parameters of the geometries (width and heights, diameters, etc.). The parameters are then re-adjusted manually to better fit the images. This process is done in Blender. The process does not really scale that well since it does require manual adjustment and parts-selection. 

I just pushed the simplified scene here: [https://github.com/KlabCMU/EmbodiedPose/tree/master/data/scenes](https://github.com/KlabCMU/EmbodiedPose/tree/master/data/scenes)
